content/en/synthetics/api_tests/udp_tests.md

```diff
@@ -0,0 +1,175 @@
+---
+title: UDP Tests
+kind: documentation
+description: 
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetic Monitoring"
+- link: "/getting_started/synthetics/api_test/"
+  tag: "Documentation"
+  text: "Get started with API tests"
+---
+## Overview
+
+UDP tests allow you to easily monitor that low-level UDP connections can be established on the ports of given hosts, ensuring availability of any services living on UDP ports. With built-in response time data, you can keep track of the performance of your network applications and be alerted in case of unexpected slowness.
+
+UDP tests can run from both [managed][1] and [private locations][2] depending on your preference for running the test from outside or inside your network.
+
+## Configuration
+
+After choosing to create an `UDP` Test, define your test's request.
```

Comment:
you can create a PR to update this for every /api_tests/xxx.md pages

Reply:
I'll commit those changes in here!